process/block/preprocess/transactionsV2_test.go

```diff
@@ -30,4 +44,1148 @@ func TestTransactionPreprocessor_SplitMiniBlockBasedOnTxTypeIfNeededShouldWork(t
 	assert.Equal(t, txHash1, mbs[1].TxHashes[0])
 }
 
-//TODO: Add more unit tests for transactionV2.go
+func TestTransactions_ApplyVerifiedTransactionShouldWork(t *testing.T) {
+	t.Parallel()
+
+	numMiniBlocks := 0
+	numTxs := 0
+	dataPool := initDataPool()
+	requestTransaction := func(shardID uint32, txHashes [][]byte) {}
+	preprocessor, _ := NewTransactionPreprocessor(
+		dataPool.Transactions(),
```

Comment:
Could you create an args struct instead of this long list of parameters?

Reply:
Done